Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM timbru31/node-alpine-git:16 as builder
+FROM timbru31/node-alpine-git as builder
 
 WORKDIR /tmp/nexus-web
 COPY . /tmp/nexus-web
 ENV GENERATE_SOURCEMAP=false
 RUN yarn && yarn --max-old-space-size=6144 build
 
-FROM node:16-alpine
+FROM node:18-alpine
 WORKDIR /opt/nexus
 COPY --from=builder /tmp/nexus-web/dist /opt/nexus
 EXPOSE 8000
```

Comment:
we can also update the entrypoint since we have the `preview` script to
`ENTRYPOINT ["npm", "run", "preview"]` since we don't have yarn installed by default in node alpine (to confirm)